llvm/lib/Target/RISCV/RISCVInsertVSETVLI.cpp

```diff
@@ -31,6 +31,7 @@
 #include "llvm/CodeGen/LiveIntervals.h"
 #include "llvm/CodeGen/LiveStacks.h"
 #include "llvm/CodeGen/MachineFunctionPass.h"
+#include "llvm/Support/ErrorHandling.h"
```

Comment:
Do we use anything from ErrorHandling.h?

Reply:
It's been removed. Thank you for the reminder. I forgot to remove it after debugging.